.github/workflows/test.yml

```
@@ -69,6 +75,23 @@ jobs:
         sed -i -e 's:~=:==:' requirements/*
         mv requirements/install.txt requirements/dev.txt
 
+    # To use libtree-sitter-bash.so installed on the system, we need to compile
+    # it manually, because ubuntu doesn't yet package (but Gentoo does)
+    - name: Install tree-sitter-bash
+      if: ${{ matrix.os == 'ubuntu-latest' && matrix.tree-sitter-bash == 'system' }}
+      run: |
+        sudo apt install build-essential wget
+        wget https://github.com/tree-sitter/tree-sitter-bash/archive/v0.19.0.tar.gz
+        tar xvzf v0.19.0.tar.gz
```

```suggestion
        tar -xvzf v0.19.0.tar.gz
```

Let's not use the ancient syntax ;-).